renderer/components/Footer.tsx

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import packageJSON from "../../package.json";
```

Comment:
Interesting that you directly imported the version from `package.json`, it's a nice idea. While it works, I think importing the whole JSON file might be a bit unnecessary.

If you take a look at index.tsx page, we're using the following snippet to fetch the app version: `setVersion(navigator?.userAgent?.match(/Upscayl\/([\d\.]+\d+)/)[1]);`. This makes sure that we're only using what we need and not importing the whole package.json (which might or might not be available during the build step).